packages/plugin-multi-tenant/src/fields/tenantsArrayField/index.ts

```diff
@@ -4,17 +4,19 @@ export const tenantsArrayField = (args: {
   arrayFieldAccess?: ArrayField['access']
   rowFields?: ArrayField['fields']
   tenantFieldAccess?: RelationshipField['access']
+  tenantFieldName: string
+  tenantsCollectionSlug: string
 }): ArrayField => ({
-  name: 'tenants',
+  name: args.tenantsCollectionSlug,
   type: 'array',
   access: args?.arrayFieldAccess,
   fields: [
     {
-      name: 'tenant',
+      name: args.tenantFieldName,
       type: 'relationship',
       access: args.tenantFieldAccess,
       index: true,
-      relationTo: 'tenants',
+      relationTo: args.tenantsCollectionSlug,
```

Comment:
I brought this change over into this [PR](https://github.com/payloadcms/payload/pull/10764).

Reply:
That fix is the error I was attempting to fix. The other stuff was just because it makes sense to also update those fields. I'll update though because we would like to not have "tenant" used at all.